lab-davidw/test/server-test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const expect = require('chai').Expect;
```

Comment:
So with this test you are going to have to use the net.connect module to turn on an instance of your server within each it block, so that you can test the commands(using .write('tcp command')). You'll also want to create an instance of a client, so that you can use it when comparing in you expect statements.